src/main/java/com/eternalcode/parcellockers/gui/ParcelListGUI.java

```diff
@@ -84,10 +78,12 @@ private void show(Player player, Page page) {
             }
 
             for (Parcel parcel : result.parcels()) {
-                List<String> newLore = this.replaceParcelPlaceholders(parcel, parcelItem.getItemStack().getItemMeta().getLore());
-                parcelItem.getItemStack().getItemMeta().setLore(newLore);
+                if (parcel.sender().equals(player.getUniqueId()) || parcel.recipients().contains(player.getUniqueId()) || player.hasPermission("parcellockers.admin.debug.seeOthersParcels")) {
```

Comment:
Consider a more readable solution.

Reply:
tbh the best thing i can do in this case is just splitting this condition multi-line

Reply:
> tbh the best thing i can do in this case is just splitting this condition multi-line

What I mean is that you can first check if the player has permission. If it is not there, you will return a loop - A more readable solution.

```suggestion
               if (!player.hasPermission("parcellockers.admin.debug.seeOthersParcels")) {
                   continue;
                }
                
                if (!parcel.sender().equals(player.getUniqueId()) {
                    continue;
                }
                
                if (!parcel.recipients().contains(player.getUniqueId()) {
                    continue;
                }
```